go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -134,14 +138,27 @@ func LaunchCluster(xtrabackup bool, streamMode string, stripes int) (int, error)
 		tablet.VttabletProcess.SupportsBackup = true
 		tablet.VttabletProcess.EnableSemiSync = true
 
+		if setupType == Mysqlctld {
+			tablet.MysqlctldProcess = *cluster.MysqlCtldProcessInstance(tablet.TabletUID, tablet.MySQLPort, localCluster.TmpDirectory)
+			tablet.MysqlctldProcess.InitDBFile = newInitDBFile
+			tablet.MysqlctldProcess.ExtraArgs = extraArgs
+			tablet.MysqlctldProcess.Password = tablet.VttabletProcess.DbPassword
+			if err := tablet.MysqlctldProcess.Start(); err != nil {
+				return 1, err
+			}
+			shard.Vttablets = append(shard.Vttablets, tablet)
+			continue
+		}
+
```

Comment:
Are we starting both mysqlctld and mysqlctl here?

Reply:
no it will start mysqlctld only